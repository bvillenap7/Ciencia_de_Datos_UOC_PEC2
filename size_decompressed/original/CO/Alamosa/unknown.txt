Light in NW sky appears to be a planet, then moves off to west at a HIGH rate of speed.;2005-09-10 21:00:00;5-10 minutes;Light in NW sky appears to be a planet, then moves off to west at a HIGH rate of speed. Camping 4.7mi. North of the Great Sand Dunes Nat. Monument on Madano Pass. Noticed light in NW sky appeared to be a large planet, stationary for several min., then moves off to west at a HIGH rate of speed.  Still visible but very faint after moving west, appeared to join a 2nd object and move in formation circling each other, just fade away.  Due to the faintness the 2nd object is not verified.;37.4725;-105.8796
I woke up with the feeling of being watched, I sat up and searched my apartment but no one was there, I then lost almost an hour of tim;2018-01-04 04:44:00;Not sure;I woke up with the feeling of being watched, I sat up and searched my apartment but no one was there, I then lost almost an hour of time I cannot recall, I only remember a tall windowless building surrounded by a strange light that was a color I cannot even describe, I am honestly scared and don't know what to think.;37.4725;-105.8796
