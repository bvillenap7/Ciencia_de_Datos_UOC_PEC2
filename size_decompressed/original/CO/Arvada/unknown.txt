Extremely fast moving silent white orbs in formation over Arvada going SW turning SSW. 9:50 pm 12/31/16;2016-12-31 09:50:00;3 seconds;Extremely fast moving silent white orbs in formation over Arvada going SW turning SSW. 9:50 pm 12/31/16 I saw a group of what looked like white orbs traveling extremely fast going SW and making a turn in formation to SSW at 9:50 pm over Arvada Colorado.  I have also seen a translucent shapeshifting boomerang craft fly over 13 times and have documented it within my own records.  This also could have been the translucent shape-shifting craft flying lower, but it looked different.  They could have been separate.  These things or thing was traveling faster than a fighter with no sound. 9:50 could be a possible route time as I have been witnessing the craft around this time after the time change.  Before the time change it was around 8:50 now 9:50?  Hmmmm flight pattern detected.....;39.8289982352941;-105.110888235294
