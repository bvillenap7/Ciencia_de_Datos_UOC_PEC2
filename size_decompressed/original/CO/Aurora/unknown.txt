Object seen shooting across the sky at a very high rate of speed and the stopped and started blinking a white light.  Object was moving;1997-12-14 20:30:00;5 Min;Summary : Object seen shooting across the sky at a very high rate of speed and the stopped and started blinking a white light.  Object was moving faster than any shooting star. The object was seen moving from the NNW at about 50 Deg. to the NW at about 35 Deg. at an incredible rate of speed.  The object then stopped immediately and began blinking a white light that did not move.  The object witnessed appeared to be a a distance of at least 50 miles.  Took this report on 12/15/97 from the witness who has been credible in the past.  The witness does not want to release her name.;39.6695444272445;-104.803991486068
7/00/98 aprox       aurora    co        looked like star           4-5 seconds               object moved slowly westward        4/7/03;1998-07-15 19:00:00;4 sec;7/00/98 aprox       aurora    co        looked like star           4-5 seconds               object moved slowly westward        4/7/03 I was sitting in my van with the side door open watching the stars.It had just turned dark and was nice out. I was perticularly focused on the big dipper when I noticed the top star (on the right) started moveing. It moved in a straight pattern to the right. It moved very slowly as though it was a very high flying plane but it couldn't be because it was just to high up. I have perfect vision and I know what I saw. After about 4 to 5 seconds it vanished. After it vanished I couldn't see the star that was supposed to be there. It struck me as being very unusual but I wrote it off to maybe I was seeing things. It started to bother me after a while and caused me to reflect on myself. After a long time I deceided that I am not crazy , I don't see things that are not there and what I saw was true.;39.6695444272445;-104.803991486068
while driving to wal-mart notice a reflection of something silvery high in the sky was to high up to notice shape or size but notice it;1998-11-03 14:30:00;5 min;while driving to wal-mart notice a reflection of something silvery high in the sky was to high up to notice shape or size but notice it didn't move like an airplane. I was on my way to do some shopping at wal-mart when I notice something very high and near the mountains, It was to high to see any detail it was only the sun reflecting of the surface that I notice. I tried to keep watch the object and drive which isn't easy. I notice an airplane that was heading towards the mountains and you could tell it was an airplane because of the white trail it left. the UFO had to be at the same altitude. Once the airliner got to close the object made a sharpe 45º turn toward the south then after a few seconds I lost sight of it. My back ground is I am 34 years old, I have a degree in Visual Communications and currently in school studing computer networking. I usually don't see strange objects in the sky. I don't know if this sighting will be helpful to your oganization but I felt like I had to tell someone. Thank you for listing.;39.6695444272445;-104.803991486068
Two fairly bright white lights moving in tandem.  ((NUFORC Note:  Witness addendum states that object was not Space Station.  PD));2007-06-19 22:00:00;4 minutes;((NUFORC Note:  ISS orbiting with Space Shuttle?  PD)) 0n 06-19-07 at approximately 10:00 PM I observed the strangest thing I had ever seen. Two fairly bright white lights moving in tandem in the northern sky above Denver Colorado. The two lights were the size and intensity of the bright star Sirius. The night was beautiful and clear, the moon was hanging large and low on the horizon. These objects or object turned southeast and flew directly over my house and then suddenly vanished.  I was standing at my open rear door of my house when the sound of an approaching helicopter caught my attention, As usual with any low flying aircraft, I would go outside to watch for it. As soon as I stepped out onto my rear deck my attention was diverted to two bright lights approaching from the north. At first I thought the lights were stars and stationary, but then I soon realized they were moving in my direction. They were moving slowly in tandem as if two planes were flying in formation, one in front of the other. This struck me as very odd so I continued to watch them. The two lights were approaching me from a slight angle and they never once deviated laterally from their position. They continued heading in my direction. I stood there trying to understand what it was that I was looking at. As it over flew my house, I ran through the house and called for my teenage son to come to the front door to observe this as well. The two of us stood watching this thing cruise overhead silently and almost effortlessly. We both stood motionless trying to understand what it was we were watching. Within a few seconds the two lights flashed off in unison and it was all over.  The object appeared to be close however, it had no discernable shape. The object made absolutely no sound. The space and distance between the two lights can be measured by extending one full arm length and turning your hand at a 90 degree angle. one hand length. The area in which this thing traveled penetrated heavily regulated airspace, (class Bravo airspace) for DIA. This portion of class B airspace has a floor of 8000 feet MSL and a ceiling of 14,500 MSL. The object was in this air space and had also flown in between Centennial (KAPA) and Buckley AFB airspace. All three are manned by radar.  I am an instrument rated Private Pilot and I think I have a pretty good understanding of what airplanes look and sound like. This was no airplane, as this object had none of the required position lights / navigation lights or strobes that one would recognize as local aviation. The object was not a small dimly lit fast moving satellite, nor was it a commonly observed meteor streaking across our celestial heavenly body, and no, it was no weather balloon. I don’t know what this thing was that my son and I observed, but it left a bad taste in our mouths. ((NUFORC Note:  Witness elects to remain totally anonymous  provides no contact information.  PD)) ((NUFORC Note:  ISS orbiting with Space Shuttle.  PD)) ADDENDUM FROM ORIGINAL WITNESS: Two fairly bright white lights moving in tandem. ((NUFORC Note: ISS orbiting with Space Shuttle? PD))  First of all, the ISS orbiting with Space Shuttle occured the the next day.  My son and I observed that as well.  What we witnessed  on 6/19/07 at 2200 Hrs was Absolutely NOT the ISS/Space shuttle.  NOT EVEN CLOSE!;39.6695444272445;-104.803991486068
0n Thursday the 29th of November 2007 between 12:34 to 12:37 pm. My friend and I were at 225 and I70 near Fitzsimons.;2007-11-29 12:34:00;1 or 2 minutes;0n Thursday the 29th of November 2007 between 12:34 to 12:37 pm.  My friend and I were at 225 and I70 near Fitzsimons.  We were heading north towards I70 when I noticed an object in the sky on the right hand side it was shimmering rather brightly, it was sparkling.  I noticed it seem metallic, also appearing to be upright in the sky, that is straight up and down position.  I only saw this for a split second, at a time, due to me driving and I didn’t want an accident.  So I told my friend about the object hoping he could see it in better detail.  I told him I have binoculars by you.  The distance I really couldn’t tell you except that it seemed about ½ the distance of altitude that jets would fly. ((NUFORC Note:  The source of this report is well known to us, and we deem her to be a quite reliable witness.  PD));39.6695444272445;-104.803991486068
Three separate craft 2 moving north to south one moving south to north over Aurora,CO;2009-07-16 22:16:00;5 minutes;Three separate craft 2 moving north to south one moving south to north over Aurora,CO I saw what appeared to be a UFO. I stuck my head inside and yelled for my husband. By the time he got out and I turned around it was gone however we saw another one that clearly was not a plane(no running lights). It was moving north to south as was the first one I saw. It got brighter(very very bright) then it got smaller and dim before disappearing. It appeared to accelerate very quickly before it moved out of sight.  I was joking to my husband and said (Well guys is that all no more?) Then another appeared moving from the south to the north and again too far away to see the shape but close enough to see that it was not a plane ( no running lights) It moved up and one could clearly see it was accelerating before it disappeared out of sight.;39.6695444272445;-104.803991486068
Craft moving from N to S later Flash in sky like star going nova;2009-09-05 20:23:00;2 min;Craft moving from N to S later Flash in sky like star going nova Saw glowing object in sky moving from due north to due south later at 21:20 saw what looked like a star going nova in the due north sky lasting only a second or two.  Checked nasa site no objects to be in view at these times.;39.6695444272445;-104.803991486068
4 objects moving and dancing at very high altitude. shiney, reflective orbs;2012-10-21 15:00:00;7-10 minutes;4 objects moving and dancing at very high altitude. shiney, reflective orbes Noticed up to 4 reflections of light at an extremely high altitude, flying around each other, would appear and disappear into the atmosphere.  Traveled across sky at high rate of speed, then would disappear again and reappear in their previous position within seconds.  Sky was clear, no cloud cover, to high for planes or birds, moved to fast for satellite, which don't fly and dance around in sky as these did.   Although no cloud cover the atmosphere appeared to separate around the objects.  The sky wad a slight shade darker blue around them.;39.6695444272445;-104.803991486068
