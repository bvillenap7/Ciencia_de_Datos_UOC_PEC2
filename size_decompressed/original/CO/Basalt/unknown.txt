fast moving, zagging bright lights and hovering at times.;2008-04-23 21:00:00;> 30 minutes;fast moving, zagging bright lights and hovering at times. It was really strange.  I was out with my dog at about 9 pm mst 04/23/08.  The sky was really clear.  I looked over toward the SW because something caught my eye. It was very bright and quite a long distance away.  I could make out red, green and white lights with my bare eyes.  It was moving very fast from side to side.  I ran in the house to get out my binoculars. Then, I could really see it.  I saw one row of horizontal very bright lights.  I went back into the house to tell tell my wife and got another set of glasses for her.  Now it had moved to mid height in the SW sky about mid height in the West.  I could not take my eyes off of it.  I was really excited and my wife was like no big deal but glad to have seen one.  You knew it was not a plane because a plane could not hover like that and make the very radical moves. I watched for about a half hour.  Toward the end a second UFO joined the first one, but quite a distance from the first one. Then, this morning on Channel 7 news they mentioned that UFO's were spotted over Phoenix.  So, someone else saw some too.   The FAA said they had nothing to report.;39.3535;-106.9988
