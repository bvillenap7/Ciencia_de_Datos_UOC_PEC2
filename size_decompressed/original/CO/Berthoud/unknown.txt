Unidentified noiseless craft slowly passing by our home at very low altitude.;2012-09-01 21:30:00;5 minutes;Unidentified noiseless craft slowly passing by our home at very low altitude. This occurred just beyond CR21. I was on the porch (open type) looking to see if there were any lightening storms that might be seen coming our way as I like to take photos of lightening.  It was cloudy but puffy clouds that had openings or space between them. Something made me look up and I saw something that looked like flat black metal coming very slowly cruising above the clouds.  I saw it for about 2 minutes as it came out of one cloud and into another.  I could not believe an airplane could fly that low and have absolutely no sound.  No sound at all and no lights unless they were on top of the ship but I didn't see any beams of light in the clouds. i thought it must have been huge because I saw a large section of this ship and there was not one bolt or screw to be seen.  Yes it was low enough that I would have seen that. It dawned on me then what it probably was.  I looked out by hanging over the porch railing to see if I could see where it was going and what shape it was.  That was when I realized it was coming back towards me, from the other end of our long porch.  I thought to myself how could such a big ship turn around that fast.  I then got very nervous and stepped away from the railing of the porch to the side of the house, waited about four minutes and looked again.  It was gone. I realized only later that duh an alien ship probably does not have any screws or bolts if their ships are of high technology like the experts say.  So it may not have been as big as I thought but it sure scared me.  I don't go out on the porch anymore if those puffy clouds with space between them are up above us.  My dog was very nervous and panting and I thought she felt a lightening storm coming. After the ship disappeared, she eventually calmed down.  I also had my camera with me but I was so floored by what I saw initially and then becoming frightened,  I didn't take any photos and have been kicking myself about that since then. ((NUFORC Note:  Witness indicates that the date of the sighting is approximate.  PD));40.2993;-105.1055
