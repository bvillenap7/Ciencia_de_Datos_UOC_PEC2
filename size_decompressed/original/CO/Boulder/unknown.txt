3 lights followed by 4 lights. triangular formation;2015-10-15 11:45:00;30 seconds;3 lights followed by 4 lights.  triangular formation 3 lights in triangle travel by with 4 other lights behind it (3 in triangle, 1 off centered);40.0217727272726;-105.258365102639
Light traveling beneath the clouds without marker lights of regular aircraft. Light looked like like it was traveling in our dorection;2016-08-15 21:15:00;30 seconds;Light traveling beneath the clouds without marker lights of regular aircraft. Light looked like like it was traveling in our dorection however light was clearly traveling on the opposite direction.;40.0217727272726;-105.258365102639
