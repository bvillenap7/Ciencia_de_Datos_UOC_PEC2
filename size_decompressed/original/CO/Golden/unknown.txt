3 lights triangular shaped, AAA fire, lights warping out, stealth-type fighter;2003-07-11 20:50:00;6-8 minutes;3 lights triangular shaped, AAA fire, lights warping out, stealth-type fighter I was at Coors Brewery taking a break on the North side of the building on some stairs.   I was just enjoying the evening light as it was fading in the mountains and scanning the horizon.  To the Northwest at 8:50pm I saw 3 lights in a triangular formation just above the mountains: the brightness was unlike a street light that has the appearance of shards radiating from it...these lights were brilliant, clear bluishwhite and were perfectly round.  The distance would not allow me to tell if there was one object or three.  I could not believe what I was seeing, just sitting there in the distance, because I had always wanted to see a UFO.   But there was no doubt in my mind, I was fixated.  I stared at them, the lights,  for about a 2 minutes and they winked out... just gone...even though I could see nothing but 3 lights I wanted to look some more.....all of a sudden, about 2-3 minutes later there were bursts of rapid firing light (similar to anti-aircraft fire, but they had no arc to them) from the ground (they were rectangular shaped and there was about 10 - 12 of them one behind the other). I could see nothing in the direction the lights were traveling but about the time the last light reached a certain height (approximately 2 inches to the right of my original sighting if you hold a ruler at arms length) there were several objects that streaked off in about a 30-45 degree angle from where I saw the AAA lights.   It was very much like watching ships in star trek going into warp,,,,it was amazing....I called to a mechanic in the room behind me to come out,  we saw nothing but there was a jet that appeared to be of stealth origins flying low over the mountains without lights heading in the general direction of my sighting the mechanic then left. I returned to work at 9:10pm. (well, I was a little late going back);39.7598265306123;-105.220002040816
Hovering lights seen in the sky from the top of Ancient Palms trail.;2014-07-04 21:45:00;~1 minute;Hovering lights seen in the sky from the top of Ancient Palms trail My friends and I walked to the top of Ancient Palms trail in Golden to see the fireworks shows of the surrounding areas.  At around 9PM the closest fireworks display in Golden began and most of us were turned in that direction watching. About 10 minutes later my friend tapped me on the shoulder and pointed in the opposite direction of the fireworks (east) and said What is that?  I turned and looked where he was pointing and saw 3 lights hovering in the sky.  They were completely stationary for about 15 seconds.  Then a fourth light appeared and they started moving slowly left to right in a perfect line. At this point my 4 friends and I were all watching and making a bit of a scene pointing and yelling in confusion and at least 10 other people had turned around and were looking too.  There were at least 30 people on top of the trail we were on, I can't say for sure how many people witnessed this object but there were several. The 4 lights appeared to be absorbed into one, and then would re appear as 4.  Then, all 4 disappeared suddenly, reappeared for a few moments, moved upwards slowly, and then disappeared again.  This all happened in less than 2 minutes. I heard one man ask one of the park rangers who were there if he saw the object and what he thought it was.  The trail ranger said hesitantly It was probably parachuters, which made no logical sense.  No one had any other suggestions of what it was and everyone appeared to have the same uneasy feeling. These were not fireworks or like anything I have ever seen in my life.  I feel confident reporting this as there were several witnesses and no explanation for these lights.;39.7598265306123;-105.220002040816
Color changing orb in the sky.;2015-04-25 22:00:00;2 hours +;Color changing orb in the sky Witnessed a color changing light above North Table Mountain.  Changed from green to red to purple and blue stay high in the sky for almost an hour then disappeared and reappeared on the other side of the mountain a minute later, lower in elevation.   Watched for another hour and saw it disappear and reappear within seconds clear across the sky on the other side of the mountain.;39.7598265306123;-105.220002040816
Strange melodic hum, higher pitched, alternating between 2 tones and moving back and forth N and W of Golden, CO.;2017-01-08 03:00:00;2.5 hours;Strange melodic hum, higher pitched, alternating between 2 tones and moving back and forth N and W of Golden, CO. Woke up at about 3 am to a loud melodic hum during the catastrophic wind storm that hit Colorado this month that sounded like it was above my house.  It was like nothing I’ve ever heard before.  It moved north away from the house and sounded like it was moving back and forth between the northern and western part of Golden.  I woke up my girlfriend and she was able to hear it to, but couldn’t make sense of it either. 	Checked around while it was going on and the sound didn’t appear to be created by the borderline record setting winds blowing through or against anything as the noise remained consistent in volume between wind gusts.  Also, being from Wyoming (arguably the windiest state in the country) and having heard a lot of wind against a lot of different things, this hum was still like nothing I’ve heard before.  I checked the skies around the house, but it was overcast and all I could see was the glow of the Denver lights bouncing off of the cloud deck.  But considering the fight or flight response that sound was eliciting from me, I’m glad I didn’t see anything if I’m honest.  My every instinct was telling me to get as far away as fast as possible.  It’s worth noting that the hum was higher pitched and seemed to alternate between 2 tones.  It scared the hell out of me but oddly the sound was kind of pretty in a way.  I’ve found a lot of reports online of a “hum” that detailed a really low pitch that almost chimes, but this was nothing like that and was both several octaves higher and constant. 	Also conferred with my next door neighbors.  They woke up from the wind at about the same time that I did and also heard the noise, but they’re pretty heavy sleepers and fell back asleep pretty quickly without giving it much attention.  To be more specific, it didn’t occur to them that they heard a sound until I brought it up a few days later.  I stayed up and listened to the hum until about 5:30am before falling back asleep myself.  As far as I can tell, no one has reported this to anyone. 	I’m prior military and a fairly level headed person.  So far, nothing has ever made the hair stand up on the back of my neck like that hum. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD)) ((NUFORC Note:  Source of report indicates that the date of the incident is approximate.  PD));39.7598265306123;-105.220002040816
